### DISCUSSION TITLE
How can I customize the Welcome Back message and add a automatic .profile source?

### DISCUSSION THREAD

**Discussion by WarpWing** [GitSquared/edex-ui#1007] [Questions] [answered]
Hello all! I've got a little question. So I was wondering how can I customize the Welcome Back message when it loads? I assume it just runs a finger command but it would be cool if I can set a custom variable for it.

![image](https://user-images.githubusercontent.com/28925758/105433766-3f935700-5c28-11eb-9d2b-93237773dd92.png)

Also, I was wondering if I could make it do a 
```bash
source .profile
```
on login as well? If anyone can assist me with that would be great :D

**Comment by GitSquared** [comment, marked as answer]
You can change your name in the welcome message by modifying the `username` [setting](https://github.com/GitSquared/edex-ui/wiki/settings.json).

As for sourcing a shell startup file, you should just use the standard method for the shell of your choice, although it has been reported that it sometimes doesn't work (see #1012, #805).
